src/helpers/snapshot.ts

```diff
@@ -65,20 +65,30 @@ export async function getProposals(address: string) {
       start_gt: now - 604800
     }
   });
+  const groupedProposals = {
+    pending: {},
+    active: {},
+    closed: {}
+  };
 
-  Object.keys(proposals).forEach(status => {
-    proposals[status].forEach(proposal => {
-      proposal.shortBody = `${removeMd(proposal.body).slice(0, 150)}…`;
-    });
+  proposals.forEach(proposal => {
+    proposal.shortBody = `${removeMd(proposal.body).slice(0, 150)}…`;
```

Comment:
This assume proposal have body, it doesn't look good when they dont or when body is less than 150 char cuz the 3 dots will be visible